scripts/sct_compute_mtr.py

```diff
@@ -86,21 +96,25 @@ def main(args=None):
     # save MTR file
     nii_mtr = nii_mt1
     nii_mtr.data = data_mtr
-    nii_mtr.save("mtr.nii")
-    # sct.run(fsloutput+'fslmaths -dt double mt0.nii -sub mt1.nii -mul 100 -div mt0.nii -thr 0 -uthr 100 mtr.nii', verbose)
+    nii_mtr.save(file_out + ".nii")
+    # sct.run(fsloutput+'fslmaths -dt double mt0.nii -sub mt1.nii -mul 100 -div mt0.nii -thr 0 -uthr 100 file_out.nii', verbose)
 
     # come back
-    os.chdir(curdir)
+    os.chdir(os.path.split(fname_mtr)[0])
 
     # Generate output files
     sct.printv('\nGenerate output files...', verbose)
-    sct.generate_output_file(os.path.join(path_tmp, "mtr.nii"), os.path.join(path_out, file_out + ext_out))
+    sct.generate_output_file(os.path.join(path_tmp, file_out + ".nii"), os.path.join(path_out, file_out + ext_out))
 
     # Remove temporary files
     if remove_temp_files == 1:
         sct.printv('\nRemove temporary files...')
         sct.rmtree(path_tmp)
 
+    # if output file location changed, notify user to move to file location
```

Comment:
is that really necessary?

Reply:
Understood, fixed in commit [bc49853](https://github.com/YangHee-Min/spinalcordtoolbox/commit/bc49853d811f41636125c67d4e8624438724d0e5)